### PR TITLE
add sha in the update server message and remove old comments

### PR DESCRIPTION
**Pull request by cpanato**
- add a sha in the comment when we update the test server
- remove old update test server messages to avoid confusion and make the PR a bit cleaner.

### CODE REVIEW

File: server/spinwick.go [hunk 1]
```diff
@@ -187,6 +188,16 @@ func (s *Server) updateSpinWick(pr *model.PullRequest) (string, bool, error) {
 		return installationID, true, errors.Wrap(err, "error waiting for build link")
 	}
 
+	// Remove old messate to reduce the amount of similar messages and avoid confusion
```
Comment (gabrieljackson) [inline server/spinwick.go]:
```suggestion
	// Remove old message to reduce the amount of similar messages and avoid confusion
```

File: server/spinwick.go [hunk 2]
```diff
@@ -226,8 +237,16 @@ func (s *Server) updateSpinWick(pr *model.PullRequest) (string, bool, error) {
 		return installationID, true, errors.Wrap(err, "encountered error waiting for installation to become stable")
 	}
 
+	// Remove old messate to reduce the amount of similar messages and avoid confusion
```
Comment (gabrieljackson) [inline server/spinwick.go]:
```suggestion
	// Remove old message to reduce the amount of similar messages and avoid confusion
```

File: server/spinwick.go [hunk 3]
```diff
@@ -226,8 +237,16 @@ func (s *Server) updateSpinWick(pr *model.PullRequest) (string, bool, error) {
...
+	if errComments == nil {
```
Comment (gabrieljackson) [inline server/spinwick.go]: It looks like we should be running `s.getComments()` before this?

File: server/spinwick.go [hunk 4]
```diff
@@ -606,3 +625,21 @@ func (s *Server) isSpinWickLabelInLabels(labels []string) bool {
 
 	return false
 }
+
+func (s *Server) removeServerUpdateOldComments(comments []*github.IssueComment, serverMessages []string, pr *model.PullRequest) {
```
Comment (gabrieljackson) [inline server/spinwick.go]: This function name is a little confusing to me. Maybe something like `removeCommentsWithSpecificMessages`?

File: server/spinwick.go (hunk 3 above)
Comment (cpanato) [reply server/spinwick.go]: we get the comments in the beginning which will be the same and also we dont use the request against github, we save the token :)

File: server/spinwick.go (hunk 4 above)
Comment (cpanato) [reply server/spinwick.go]: sounds good

File: server/spinwick.go (hunk 3 above)
Comment (gabrieljackson) [reply server/spinwick.go]: Okay!

**Pull request merged**